src/DonorProfiles/Helpers/FormatList.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Give\DonorProfiles\Helpers;
+
+/**
+ * Formats an associative array into a JS parsable array of objects.
+ *
+ * @since 2.8.0
+ */
+class FormatList {
```

Comment:
If no changes have been made, then we can probably extract this for re-use - I'll look into that.

Reply:
@henryholtgeerts https://packagist.org/packages/kjohnson/format-object-list

Reply:
Thanks for making that package available! Done, ref: https://github.com/impress-org/givewp/pull/5486/commits/1e56795ca28d972bcf2539e48bcae9c6b8626ee5